pkg/snapstore/abs_snapstore.go

```diff
@@ -90,7 +91,7 @@ func getCredentials(prefixString string) (string, string, error) {
 		return credentials.StorageAccount, credentials.SecretKey, nil
 	}
 
-	if dir, isSet := os.LookupEnv(prefixString + absCredentialFile); isSet {
+	if dir, isSet := os.LookupEnv(prefixString + absCredentialDirectory); isSet {
 		credentials, err := readABSCredentialFiles(dir)
```

Comment:
Pretty bad names. You can leave as is right now as it is old code. Function name `readABSCredentialFiles` just says to read the file but you are reading the file and also initializing an instance of `absCredential` from it and this initialization will be different when input is a JSON file vs a bunch of files in a directory.

All of this will be changed in steward. Intention to record this comment for you is to tell you that any new code that you write should have a clarity of purpose w.r.t any function, field, interface type etc that is introduced and that purpose should clearly reflect in names.

Reply:
Understood. Thanks for the explanation.